packages/trpc/react/trpc.ts

```diff
@@ -1,13 +1,13 @@
 import superjson from "superjson";
 
-import { httpBatchLink } from "../client/links/httpBatchLink";
 import { httpLink } from "../client/links/httpLink";
 import { loggerLink } from "../client/links/loggerLink";
 import { splitLink } from "../client/links/splitLink";
 import { createTRPCNext } from "../next";
 // ℹ️ Type-only import:
 // https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export
 import type { TRPCClientErrorLike } from "../react";
+import { httpBatchLink } from "../react";
```

Comment:
Is this intended?

Reply:
It would have been a mistake from testing performance, should have also come from client/links